### DISCUSSION TITLE
Admin UI 1.8.1: Product Description Widget is not visible

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1852] [Q&A]
Upgraded Vendure server from 1.7.4 to 1.8.1. After that, I cannot see the Description widget in the Product Add & Edit screens. And also noticed a console error.
<img width="1430" alt="Screenshot 2022-10-29 at 10 20 49 AM" src="https://user-images.githubusercontent.com/83760807/198814527-aefd336c-5412-4aee-b751-7a13e3e9297e.png">

**Comment by michaelbromley** [comment]
If anyone else runs into this: try deleting node_modules & your lockfile and re-installing.

Looks like some dependency issue with the Prosemirror editor packages.
